go.mod

```diff
@@ -17,6 +17,12 @@ require (
 	k8s.io/klog/v2 v2.80.1
 	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
 	sigs.k8s.io/controller-runtime v0.13.0
+	sigs.k8s.io/controller-tools v0.10.0
```

Comment:
[v0.11.1](https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.11.1) if we use the latest version here we can then easily bump to `ENVTEST_K8S_VERSION = 1.26` when we'll do the repo's version bump.

Reply:
Shall I bump to 1.26 within this PR then?

Reply:
sigs.k8s.io/controller-tools v0.11.1

Reply:
Doesn't look like this comment has been resolved?

Reply:
Resolved.